integration/storyshots.test.js

```diff
@@ -7,13 +7,14 @@ const getMatchOptions = () => ({
   failureThresholdType: "percent",
 });
 
+//This is needed to keep CI from failing due to viewport differences
 const view = {
-  name: "Desktop 800x600",
-  userAgent: "placeholder",
+  name: 'Desktop 1200x800',
+  userAgent: 'placeholder',
   viewport: {
-    width: 800,
-    height: 600,
-  },
+      width: 1200,
+      height: 800
+  }
```

Comment:
Found that these dimensions led to less false failures in travis in fundamental-react